samples/model-builder/deploy_model_sample.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import aiplatform
+from typing import Optional, Sequence, Tuple, Dict
+from google.cloud.aiplatform import explain
+
+#  [START aiplatform_sdk_deploy_model_sample]
+def deploy_model_sample(
+    project,
+    location,
+    model_name: str,
+    endpoint: Optional["Endpoint"] = None,
+    deployed_model_display_name: Optional[str] = None,
+    traffic_percentage: Optional[int] = 0,
+    traffic_split: Optional[Dict[str, int]] = None,
+    min_replica_count: int = 1,
+    max_replica_count: int = 1,
+    metadata: Optional[Sequence[Tuple[str, str]]] = (),
+    sync: bool = True,
+):
+
+    aiplatform.init(project=project, location=location)
+
+    model = aiplatform.Model(model_name=model_name)
+
+    # AutoML models require a machine_type of None
```

Comment:
@vinnysenthil do all AutoML Models require automatic resources?

Reply:
Actually, we talked about this and it's a little unclear what the truth of this is.

We're still trying to find out the answer from the predict team.

Reply:
@sasha-gitg I talked to prediction team and unfortunately, they do not have the answer. I will remove the comment for now and update this if I find out more.